kong-ingress-data-plane-postgres.yaml

```diff
@@ -75,5 +79,13 @@ spec:
     port: 443
     targetPort: 8443
     protocol: TCP
+  - name: http
+    port: 80
```

Comment:
Port 80 is being bound to multiple target ports, once for Proxy side and once for Admin API, that doesn't look right?

Reply:
agreed fixed on the parent branch f0d9f53be25e49a1c1cef1b84eb5b1a9014e137a